### PR TITLE
Migrate formats for Employment Tribunal Decisions

### PR DESCRIPTION
**Pull request by Rosa-Fox**
https://trello.com/c/PT4A0cdJ/540-et-decisions

### CODE REVIEW

File: spec/models/employment_tribunal_decision_spec.rb [hunk 1]
```diff
@@ -0,0 +1,185 @@
...
+      "public_timestamp" => "2015-11-16T11:53:30+00:00",
+      "tribunal_decision_categories" => ["age-discrimination"],
+      "tribunal_decision_country" => "england-and-wales",
+      "tribunal_decision_decision_date" => "2015-07-30",
+      "organisations" => ["employment-tribunal"],
+    }
+  }
+
+  before do
+    @fields = [
```
Comment (boffbowsh) [inline spec/models/employment_tribunal_decision_spec.rb]: This and `@employment_tribunal_decisions` could be `let`s too I think.

File: spec/models/employment_tribunal_decision_spec.rb [hunk 2]
```diff
@@ -0,0 +1,185 @@
...
+    publishing_api_has_fields_for_format('specialist_document', @employment_tribunal_decisions, @fields)
+
+    @employment_tribunal_decisions.each do |decision|
+      publishing_api_has_item(decision)
+    end
+
+    Timecop.freeze(Time.parse("2015-12-18 10:12:26 UTC"))
+  end
+
+  context "#all" do
```
Comment (boffbowsh) [inline spec/models/employment_tribunal_decision_spec.rb]: It's better to use `describe` for blocks describing the behaviour of a method, then `context` to define the context for a particular set of behaviours for that method/object.

http://betterspecs.org/#describe

File: spec/models/employment_tribunal_decision_spec.rb (hunk 1 above)
Comment (tommyp) [inline spec/models/employment_tribunal_decision_spec.rb]: What's the correct syntax for turning an iteration into a `let` block?

Comment (boffbowsh) [inline spec/models/employment_tribunal_decision_spec.rb]:
```ruby
let(:fields) { %i[base_path content_id] }
let(:employment_tribunal_decisions) { 10.times.map { |n| employment_tribunal_decision_content_item(n) } }
```

Comment (tommyp) [inline spec/models/employment_tribunal_decision_spec.rb]: Ah yeah, I had a bit of a brain fart there.

File: spec/models/employment_tribunal_decision_spec.rb (hunk 2 above)
Comment (Rosa-Fox) [inline spec/models/employment_tribunal_decision_spec.rb]: Thanks for the feedback, I have altered and pushed.

**Pull request merged**